projects/ccd-case-ui-toolkit/src/lib/shared/components/search-result/search-result.component.html

```diff
@@ -35,12 +35,14 @@ <h2 class="heading-h2" id="search-result-heading__text" tabindex="-1">{{ caseSta
       </th>
       <th *ngFor="let col of resultView.columns">
         <table class="search-result-column-header"
-          attr.aria-label="Sort by {{col.label}} {{ isSortAscending(col)? 'ascending' : 'descending' }}">
+          [attr.aria-label]="'Sort by ' + col.label + ' ' + isSortAscending(col)? 'ascending' : 'descending'">
```

Comment:
@MateiHMCTS - is it not possible to retain the `{{ }}` interpolation instead of using the concatenation operator?